packages/Webkul/Admin/src/Resources/views/catalog/products/accordians/variations.blade.php

```diff
@@ -124,7 +124,7 @@ class="control"
                 <tbody>
 
                 <variant-item v-for='(variant, index) in variants' :variant="variant" :key="index"
-                              :index="variant.id"
+                              :index="index"
```

Comment:
Here `index` can be added but you need to look at here also and some more cleanup s needed,
https://github.com/bagisto/bagisto/blob/36a6ff707750e0983e951b9f892c4fdec428d6fc/packages/Webkul/Product/src/Type/Configurable.php#L116